package.json

```
@@ -7,7 +7,10 @@
     "dev": "next dev",
     "build": "next build",
     "start": "next start",
-    "lint": "next lint"
+    "lint": "next lint",
+    "test": "jest",
+    "test:ci": "jest --ci --coverage",
```

`--passWithNoTests`がないと、テストケースがない場合にCIが落ちると思います。

```suggestion
    "test": "jest --passWithNoTests",
    "test:ci": "jest --ci --coverage --passWithNoTests",
```

---

CI のスクリプトはまだ追加してないのでなくてもいいと思いますよ
テストケース一個追加したら CI でもテスト実行する GitHub Workflows 書こうと思います